src/locale/lang/th-TH.js

```diff
@@ -0,0 +1,99 @@
+export default {
+    i: {
+        locale: 'th-TH',
+        select: {
+            placeholder: 'ตัวเลือก',
+            noMatch: 'ข้อมูลไม่ตรงกัน',
+            loading: 'ดาวน์โหลด'
+        },
+        table: {
+            noDataText: 'ไม่พบข้อมูล',
+            noFilteredDataText: 'ไม่พบตัวกรองข้อมูล',
+            confirmFilter: 'ยืนยัน',
+            resetFilter: 'รีเซ็ต',
+            clearFilter: 'ทั้งหมด'
+        },
+        datepicker: {
+            selectDate: 'เลือกวัน',
+            selectTime: 'เลือกเวลา',
+            startTime: 'เริ่มเวลา',
+            endTime: 'สิ้นสุดเวลา',
+            clear: 'ล้างข้อมูล',
+            ok: 'ตกลง',
+            datePanelLabel: '[ดดดด] [ปปปป]',
```

Comment:
@NatJNP sorry, maybe I was unclear. This line should follow the syntax here: https://github.com/iview/iview/blob/2.0/src/components/date-picker/util.js#L67-L74

You can see for example the [`zh-CN` file](https://github.com/iview/iview/blob/2.0/src/locale/lang/zh-CN.js#L25) to help.